__tests__/app.test.js

```diff
@@ -159,3 +159,85 @@ describe("GET /api/reviews/:review_id/commnets", () => {
       });
   });
 });
+//
+describe("POST /api/reviews/:review_id/commnets", () => {
+  it("201: should accept object with username and body and respond with posted comment", () => {
+    const review_id = 1;
+    const newComment = {
+      username: "mallionaire",
+      body: "what an awesome comment",
+    };
+    return request(app)
+      .post(`/api/reviews/${review_id}/comments`)
+      .send(newComment)
+      .expect(201)
+      .then(({ body }) => {
+        const { comment } = body;
+        expect(comment).toHaveProperty("comment_id", 7);
+        expect(comment).toHaveProperty("votes", expect.any(Number));
+        expect(comment).toHaveProperty("created_at", expect.any(String));
+        expect(comment).toHaveProperty("author", "mallionaire");
+        expect(comment).toHaveProperty("body", "what an awesome comment");
+        expect(comment).toHaveProperty("review_id", review_id);
+      });
+  });
+  //
+  it("400: should respond with 400 bad request if review id is not a number", () => {
+    const review_id = "banana";
+    const newComment = {
+      username: "mallionaire",
+      body: "what an awesome comment",
+    };
+    return request(app)
+      .post(`/api/reviews/${review_id}/comments`)
+      .send(newComment)
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("bad request");
+      });
+  });
+  //
+  it("400: should respond with 400 if new comment has not got right keys", () => {
```

Comment:
I think you cane more specific in the test description here, its only checking really to see if a body key is missing, I would probably say as much :)